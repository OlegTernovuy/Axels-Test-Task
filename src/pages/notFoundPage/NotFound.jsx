import React from 'react';

const NotFound = () => (
  <div>This page does not exist</div>
);

export default NotFound;
