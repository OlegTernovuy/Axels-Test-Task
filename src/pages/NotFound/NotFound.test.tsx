import renderer from 'react-test-renderer';

import NotFound from './NotFound';

describe('NotFound page snapshot component', () => {
    it('Matches Snapshot', () => {
        const NotFoundPage = renderer.create(<NotFound />).toJSON();
        expect(NotFoundPage).toMatchSnapshot();
    });
});
