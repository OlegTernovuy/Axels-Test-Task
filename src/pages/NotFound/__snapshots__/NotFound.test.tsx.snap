// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound page snapshot component Matches Snapshot 1`] = `
<div>
  This page does not exist
</div>
`;
